doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
 
 
-Problem: Reduced scalability 
+Problem 2: Reduced scalability 
 ----------------------------
 
-Integrating kernel libraries with frameworks is increasingly becoming 
-nontrivial due to the growing number of new deep learning accelerators. 
-For each new deep learning accelerator, a custom kernel library integration 
-must be implemented by a team of experts. This labor-intensive work is 
-further amplified if you want your DL accelerator to support a number of 
-different frameworks. The work must be revisited any time you upgrade or 
-expand your network's hardware. Each integration is unique to the framework 
-and its set of deep learning operators, its view on memory layout, its 
-feature set, etc.
+Due to the growing number of new deep learning accelerators, integrating kernel libraries with frameworks has become increasingly nontrivial. For each new deep learning accelerator, a custom kernel library integration must be implemented by a team of experts. This labor-intensive work is further complicated by the number of frameworks as illustrated in the diagram below using orange lines. 
 
+.. _figure-B:
+
+.. figure:: ../graphics/lorem-ipsum.png
+   :width: 555px
+   :alt: 
 
-nGraph solves this problem with nGraph bridges. A bridge takes a computational 
-graph and reconstructs it in the nGraph IR with a few primitive nGraph 
-operations. With the unified computational graph, kernel libraries no longer 
-need to be separately integrated to each deep learning framework. Instead, the 
-libraries only need to support nGraph primitive operations, and this approach 
-streamlines integration process for the backend.  
+Each framework must be manually integrated with each hardware-specific kernel library. Additionally, each integration is unique to the framework and its set of deep learning operators, view on memory layout, feature set, etc. Each connection that needs to be made increases the amount of work, resulting in a fragile setup that is costly to maintain.
 
+nGraph solves this problem with nGraph bridges. A bridge takes a computational graph from a supported framework and reconstructs it in the nGraph IR with a few primitive nGraph operations. With a unified computational graph, kernel libraries no longer need to be separately integrated into each deep learning framework. Instead, the ibraries only need to support nGraph primitive operations, and this approach streamlines the integration process for the backend.  
```

Comment:
`computational graph from a supported framework`: not all frameworks use graphs as their native formalisms... it's a bit wordy, but maybe `a computational graph or similar structure`

Reply:
Typo: `ibraries`